### PR TITLE
Adding few overlays, renaming myself, changing description

### PR DESCRIPTION
**Pull request by msva**
refs:
8a262a5: https://bugs.gentoo.org/show_bug.cgi?id=597860
3f52573: https://bugs.gentoo.org/show_bug.cgi?id=597862

### CODE REVIEW

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -905,8 +906,9 @@ FIN
   </repo>
   <repo quality="experimental" status="unofficial">
     <name>crossdev</name>
-    <description lang="en">Fixed ebuilds for uclibc and other embedded systems</description>
     <homepage>https://github.com/alphallc/crossdev/</homepage>
+    <description lang="en">Ebuilds for embedded systems</description>
+    <description lang="en">Пакеты для встраиваемых систем</description>
```
Comment (maksbotan) [inline files/overlays/repositories.xml]: ru

Comment (msva) [inline files/overlays/repositories.xml]: yeah, sorry.

Fixed.

File: files/overlays/repositories.xml [hunk 2]
```diff
@@ -110,10 +110,11 @@ FIN
   <repo quality="experimental" status="unofficial">
     <name>activehome</name>
     <description lang="en"><![CDATA[Gentoo overlay for ebuilds related to 'hand-made' Active Home construction]]></description>
+    <description lang="en"><![CDATA[Оверлей для ebuild'ов связанных созданием самодельного умного дома]]></description>
```
Comment (mgorny) [inline files/overlays/repositories.xml]: Wrong lang=""

File: files/overlays/repositories.xml [hunk 3]
```diff
@@ -905,8 +906,9 @@ FIN
   </repo>
   <repo quality="experimental" status="unofficial">
     <name>crossdev</name>
-    <description lang="en">Fixed ebuilds for uclibc and other embedded systems</description>
     <homepage>https://github.com/alphallc/crossdev/</homepage>
+    <description lang="en">Ebuilds for embedded systems</description>
```
Comment (mgorny) [inline files/overlays/repositories.xml]: Please don't change the order of elements, the old DTD was kinda picky about stuff like that, and some tools may actually rely on it still.

File: files/overlays/repositories.xml (hunk 1 above)
Comment (mgorny) [inline files/overlays/repositories.xml]: lang=""

File: files/overlays/repositories.xml (hunk 3 above)
Comment (msva) [inline files/overlays/repositories.xml]: 1) talking about wrgong "lang" - looks like you're reviwed kida outdated commit.
2) And, talking about changed order — sorry, I did it accidentally. Fixed now.
// although, xmllint said nothing about that violation :-/

**Comment by msva** [comment]
@mgorny fixed :)

**Pull request merged**